.gitignore

```diff
@@ -1,7 +1,6 @@
 # generated files
-i18n_generated.js
-i18n_generated.d.ts
-i18n_generated.d.ts.map
+i18n_generated*
+icon-generated*
```

Comment:
I don't think it's good to ignore these generated files. Ignoring them makes switch among branches painful.

They are generated by tools indeed, but they are also source code. I would prefer our code can run without building  scripts.

Beyond that, we can also track the **real** changes.

Reply:
if we do this, we should also track i18n generated files. We can discuss that and (if we decided to) make a new PR to track both icons generated and i18n generated.

I'll not do that in this PR.